### ISSUE TITLE
How to deploy my own tiny-tflite model, please enlighten me, thank you!（如何对我自己的tflite模型进行部署，还望赐教，谢谢！）

### ISSUE THREAD

**Issue by czy9987** [hunglc007/tensorflow-yolov4-tflite#204]
I have implemented the author's original yolov4-416-fp32.tflite to run on android, but when I replaced the original tflite and label files, the program could not run.

I made the above replacement twice:
Once I used the yolov4-tiny model downloaded here (https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-tiny.weights) and converted it to a tflite file, replacing MainActivity and TF_OD_API_MODEL_FILE and TF_OD_API_LABELS_FILE in YoloV4Classifier, the program can be opened normally, but when I click DETECT or CAMERA, the program will close;
Another time I used the yolov4-tiny model I trained to perform the above replacement. At this time, the program could not be opened directly, and it would crash when it was not opened.

other:
My own yolov4-tiny model has made some modifications to the author's dataset.py, mainly by deleting the settings of the L type border and other information;
I also modified YOLO.ANCHORS_TINY in config.py, I used the k-means algorithm to get six anchors from my training data, and replaced the original data with them

End: My English level is limited, so what I said may not be professional. Please forgive me. I hope the seniors who implement their own tflite model deployment can give pointers, thank you!

-------------------------------------------------------------------------------------------------------------------------------------------
我已经实现了作者原本的yolov4-416-fp32.tflite在android上的运行，但是当我替换掉原本的tflite和label文件后，程序无法运行。

我进行两次上述替换：
一次是用 此处（https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-tiny.weights）下载的yolov4-tiny模型，将其转为tflite文件后，分别替换了 MainActivity 和 YoloV4Classifier中的 TF_OD_API_MODEL_FILE 和 TF_OD_API_LABELS_FILE ，程序可以正常打开，但是当我点 DETECT 或者 CAMERA 时程序就会关闭；
另一次时使用我自己训练的yolov4-tiny模型进行以上替换，这时程序直接无法打开，没得打开时都会闪退。

其他：
我自己的yolov4-tiny模型是对作者的 dataset.py 进行了一些修改，主要是删除了其中的 L 类边框等信息的设置；
还修改了config.py 中的 YOLO.ANCHORS_TINY ，我利用k-means算法从我的训练数据中得到了六个anchor，并用他们替换了原本的数据

结尾：我的英文水平有限，因此说的话可能不太专业，还请见谅，希望实现自己的tflite模型部署的前辈进行指点，谢谢！
